custom/enikshay/management/commands/get_users_for_bets.py

```diff
@@ -118,6 +118,7 @@ def _is_relevant_location(location):
             return
 
         user_data = BETSUserPayloadGenerator.serialize(self.domain, user)
+        user_data['migrationCrtdRecordId'] = user.user_data.get('agency_id_legacy', '')
```

Comment:
are they aware that this won't be set on agencies that were created in eNikshay? 
I think they might be looking for something else here?